modules/KalturaSupport/components/dualScreen/dualScreen.js

```diff
@@ -405,6 +413,7 @@
 
                         var _this = this;
                         this.initView();
+						this.hideEmbeddedCaptionsFromSecondary();
```

Comment:
fix the indentation. 
also, if you move it after `initControlBar` so you can keep the logic like was before by using `canManipulateControlViews`

Reply:
do you think we should still use canManipulateControlViews function?.. how hiding the embedded captions is related to controlBar or if the view mode is locked?

Reply:
I said you can not you should. if you want to make the change as small as possible

Reply:
@yairans i opened the github editor and the indentation doesn't exist there, nor in my webstorm... seems to be only here.